centreon/src/Core/Domain/Common/GeoCoords.php

```diff
@@ -40,8 +40,8 @@ class GeoCoords implements \Stringable
     private const REGEX_FULL = '/^' . self::REGEX_LATITUDE . ',\s*' . self::REGEX_LONGITUDE . '$/';
 
     /**
-     * @param numeric-string $latitude
-     * @param numeric-string $longitude
+     * @param string $latitude
+     * @param string $longitude
```

Comment:
We must be discussed this because it breaks theorically the object.

👉🏻 We may want to split the construct declaration and property declaration.